doc/userguide/output/eve/eve-json-output.rst

```diff
@@ -70,6 +70,21 @@ Alerts are event records for rule matches. They can be amended with
 metadata, such as the application layer record (HTTP, DNS, etc) an
 alert was generated for, and elements of the rule.
 
+The alert is amended with application layer metadata for signatures
+using application layer keywords. It is also the case for protocols
+over UDP as each single packet is expected to contain a PDU.
```

Comment:
@inashivb is this the case for DCERPC/UDP with fragments ?

Is DCERPC/UDP with fragments = multiple packets but only one tx for the reassembled stuff ?